releases/2.12.14.md

```diff
@@ -3,7 +3,8 @@ Scala 2.12.14:
     ([#9589](https://github.com/scala/scala/pull/9589), [#9595](https://github.com/scala/scala/pull/9595) and [#9620](https://github.com/scala/scala/pull/9620) by [@smarter](https://github.com/smarter) and [@neko-kai](https://github.com/neko-kai))
   * Reduces lock contention between compilers when reading classpath JAR files. ([#9549](https://github.com/scala/scala/pull/9549) by [@retronym](https://github.com/retronym))
   * Propagate custom `TaskSupport` through operations of parallel collections. ([#9529](https://github.com/scala/scala/pull/9529), backport of fix for [scala/scala-parallel-collections#152](https://github.com/scala/scala-parallel-collections/issues/152), by [@lrytz](https://github.com/lrytz)) 
-  * Update version of ASM dependency, enabling interop with Java 17-compiled classes. ([#9488](https://github.com/scala/scala/pull/9488) by [@sethtisue](https://github.com/sethtisue))
+  * upgrade to asm 9.1, for JDK 17 support ([#9488](https://github.com/scala/scala/pull/9488) by [@SethTisue](https://github.com/SethTisue))
+  * Fix cyclic error in runtime reflection (protobuf), a regression that [prevented Spark upgrading](https://github.com/apache/spark/pull/31223) to 2.12.13  ([#9478](https://api.github.com/repos/scala/scala/pulls/9478) by [@retronym](https://github.com/retronym))
```

Comment:
This is the json api url instead of the web url.

Reply:
@retronym can you update https://github.com/scala/scala/releases/tag/v2.12.14 for this?

Reply:
Thanks for catching that. I've fixed two incorrect URLs in the release notes.